source/extensions/filters/network/kafka/protocol_code_generator/kafka_request_resolver_cc.j2

```diff
@@ -0,0 +1,28 @@
+// DO NOT EDIT - THIS FILE WAS GENERATED
```

Comment:
Where are you generating this code? I don't think this should be necessary? Can you take a look at where we generate protobuf files to? I think those go into Bazel internal directories so we shouldn't need to worry about these type of comments or clang-format? Also, same comment about comments on the template file.